radis/lbl/factory.py

```diff
@@ -1197,6 +1217,152 @@ def eq_spectrum_gpu(
+
+        Returns
+        -------
+        s : TYPE
+            DESCRIPTION.
+
+        Examples
+        --------
+        ::
+            from radis import SpectrumFactory
+            from radis.tools.plot_tools import ParamRange
+
+
+            sf = SpectrumFactory(2200, 2400, # cm-1
+                              molecule='CO2',
+                              isotope='1,2,3',
+                              wstep=0.002,
+                              )
+
+            sf.fetch_databank('hitemp')
+
+            s = sf.eq_spectrum_gpu_explore(Tgas=ParamRange(300.0,2000.0,1200.0), #K
+                                           pressure=0.2, #bar
+                                           mole_fraction=0.1,
+                                           path_length=ParamRange(0,10,2.0), #cm
+                                           slit_FWHM=ParamRange(0,1,0), #cm
+                                           emulate=False,
+                                           )
+        .. minigallery:: radis.lbl.SpectrumFactory.eq_spectrum_gpu_explore
+            :add-heading:
+
+        """
+
+        import matplotlib.pyplot as plt
+        from matplotlib.widgets import Slider
+
+        try:
+            from radis.lbl.gpu import gpu_iterate
+        except (ModuleNotFoundError):
+            print("Failed to load GPU module, exiting!")
+            exit()
+
+        self.sliders = {}
+
+        for key in kwargs:
+            if is_range(kwargs[key]):
+                self.sliders[key] = kwargs[key]
+                kwargs[key].name = key
+                kwargs[key] = kwargs[key].valinit
+
+        s = self.eq_spectrum_gpu(*vargs, **kwargs)
+
+        if is_range(slit_FWHM):
+            self.sliders["slit_FWHM"] = slit_FWHM
+            slit_FWHM.name = "slit_FWHM"
+            slit_FWHM = slit_FWHM.valinit
+
+        s.apply_slit(slit_FWHM, unit="cm-1")  # to create 'radiance', 'transmittance'
+        s.conditions["slit_FWHM"] = slit_FWHM
+
+        line = s.plot(var, show=True, **plotkwargs)
+        fig = plt.gcf()
+
+        def update_plot(val):
+            abscoeff, transmittance, iter_params = gpu_iterate(
+                s.conditions["pressure_mbar"] * 1e-3,
+                s.conditions["Tgas"],
+                s.conditions["mole_fraction"],
+                l=s.conditions["path_length"],
+                slit_FWHM=s.conditions["slit_FWHM"],
+                verbose=False,
+                gpu=(not s.conditions["emulate_gpu"]),
+            )
+            if var == "abscoeff":
+                new_y = abscoeff
+            elif var == "transmittance":
+                new_y = transmittance
+            elif var in ["emissivity", "radiance"]:
+                emissivity = 1 - transmittance
+                if var == "emissivity":
+                    new_y = emissivity
+                else:  # var = 'radiance':
+                    # TODO: Is this a real thing - radiance with slit function?
```

Comment:
"Is this a real thing - radiance with slit function?"

Yes, it's the typical output for OES spectra. In RADIS it's created after using [s.apply_slit()](https://radis.readthedocs.io/en/latest/source/radis.spectrum.spectrum.html#radis.spectrum.spectrum.Spectrum.apply_slit)   or directly [convolve_with_slit(w, I, w_slit, I_slit, ...)](https://radis.readthedocs.io/en/latest/source/radis.tools.slit.html#radis.tools.slit.convolve_with_slit)


Applying a slit is not so obvious : the slit unit may be different from the spectrum default unit (in OES slits are measured in "nm", but Radis LBL computes in cm-1)  ; in dispersive spectroscopy the instrumental function may even change with the wavelength, because the grating reciprocal dispersion function changes  (in which case you have to apply the slit function "by-part").

That's why apply_slit exists and does all the work. 

In the future we want this to happen in the GPU to be able to do fitting ; but in a first approach I think it would be safer/more robust to have the GPU compute the spectrum only ; and let the CPU apply the slit with apply_slit.

Something like : 
```python
from radis.spectrum.utils import CONVOLUTED_QUANTITIES
if var in CONVOLUTED_QUANTITIES:
    s.update(var+"_noslit")    # gets the spectral array, more on this later
    s = s.take(var+"_noslit").apply_slit(...)   # only apply the slit to the final array 
else:
    s.update(var)      # gets the required spectral array 
```

Use-case is related to the following request : #245